### ISSUE TITLE
Possible glitch with spaces around blocks

### ISSUE THREAD

**Issue by vitexikora** [jballant/webpack-strip-block#10]
This block:
```js
export default /* DEV */ Object.keys(/* /DEV */ actionTypes /* DEV */).reduce(
```
should translate to ```export default actionTypes``` on production, however it goes to ```export defaultactionTypes```, no matter how many spaces I add there. Only a newline is helping. 

Is this a bug?

Webpack loader:
```js
module: {
    rules: [
      {
        test: /\.jsx?$/,
        include: path.join(__dirname, 'src'),
        use: [
          'babel-loader',
          'webpack-strip-block?start=DEV&end=%2FDEV'
        ]
      },
...
```

**Comment by jballant** [comment]
I didn't really intend the plugin to be used on a single line, so it assumes that it should remove all the space around the tag until it hits a new line.

You might be able to write it with newlines for the time being:
```js
const actionTypes =
/* DEV */
    Object.keys(
/* /DEV */
        actionTypes
/* DEV */
    ).reduce(function (memo, val) {
        // reducing
    });
/* /DEV */
export default actionTypes;
```
I realize that is probably not ideal, but until I can decide how to handle this kind of scenario, that might be a workaround.

**Comment by vitexikora** [comment]
I see,
well my opinion is that this behavior (removing whitespace outside the
block) is very counter-intuitive (you would logically expect only the
inside of the block to dispappear) and also makes it less universal: cannot
work inline, and the disadvantage of making it work correctly would only be
leaving a blank line, which won't break your app (on production, when you
dont expect it). Luckily this error was a syntax one, so the app failed to
build, instead of being deployed with a runtime error.
Please consider changing this behavior (maybe by introducing a config
switch to keep BC).
I have to say that Im a big fan of this neat webpack loader, I was looking
for something like this from the time I started with JS apps, now Im using
it on like 6 projects and forced my colleagues to use it as well :)

**Comment by jballant** [comment]
I agree it is a problem that the behavior isn't intuitive. It was done that way originally to make the output cleaner if you used individual lines for the start and end blocks, but there should at least be an option to turn it on/off. I might add an option for `assumeInlineComments: false` or something similar that you can turn off. Or I could just change it to never remove space and the output bundle will just have unnecessary spacing.

**Comment by vitexikora** [comment]
Nice!
I think that both solutions are good, I would go with the 

> Or I could just change it to never remove space and the output bundle will just have unnecessary spacing.

And maybe add an option `removeOuterWhitespace: bool` defaulting to false, but I can't imagine someone would use it (as removing whitespace is a task for a minifier and one is bundled with webpack). And deploy as 0.3.0 as it might be a big change for some usage. Extra whitespace shouldnt be a problem anywhere, unless someone was using it in a crazy way such as:
```
const someValue = 1
const someDevValue = 2
return some /* DEV */ Dev /* /DEV */ Value
```
This would indeed break, but it is crazy, no linter / syntax highligher would let you do this.

**Comment by jballant** [comment]
https://github.com/jballant/webpack-strip-block/pull/12
